gremlin-dotnet/test/Gremlin.Net.IntegrationTest/Process/Traversal/DriverRemoteConnection/GraphTraversalTests.cs

```diff
@@ -330,7 +330,7 @@ public void shouldUseMaterializedPropertiesTokenInVP()
             {
                 Assert.NotNull(vp);
                 // GraphSON will deserialize into null and GraphBinary to []
-                Assert.True(vp.Properties == null || vp.Properties.Length == 0);
+                Assert.True(vp.Properties.Length == 0);
```

Comment:
nit: can be simplified to something like `Assert.Empty(vp.Properties)`, same below